README.md

```diff
@@ -31,6 +31,11 @@ Kinto-based systems.
 
 NodeJS v4+ and npm 2.14+ should be installed and available on your machine.
 
+Various pre-commit hooks are available to help prevent you from
+pushing sub-optimal code; to use these, ``pip install --user
```

Comment:
Well, the tool is written in Python, so one way or another, you'll have to install Python. It seems that pre-commit can be installed itself using `brew`, but not everyone uses OS X, and I guess there's a `curl` installation option as well. I didn't worry about it too much because this is a tool for administering Kinto, which is itself written in Python, so I think there's a good chance you'll have Python and even `pip` available on your machine.